Linting fails with typescript

When using eslint with a project in typescript it fails when there is a file with only types.

I have made the smallest repro here https://github.com/zanza00/single-spa-ts-eslint

The error is this.

```
yarn run v1.22.4
warning package.json: No license field
$ eslint src --ext js,ts,tsx

/Users/spicciani/dev/microfrontend/single-spa-ts-eslint/src/random/types.ts
  1:13  error  'RandomString' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/packages/generator-single-spa/src/common-templates/typescript/package.json
+++ b/packages/generator-single-spa/src/common-templates/typescript/package.json
@@ -1,7 +1,7 @@
 {
   "devDependencies": {
     "@babel/preset-typescript": "^7.9.0",
-    "eslint-config-ts-react-important-stuff": "^2.0.0",
+    "eslint-config-ts-important-stuff": "^1.0.0",
     "typescript": "^3.8.3",
     "webpack-config-single-spa-ts": "^1.4.2",
     "ts-config-single-spa": "^1.7.0"

--- a/packages/generator-single-spa/src/react/generator-single-spa-react.js
+++ b/packages/generator-single-spa/src/react/generator-single-spa-react.js
@@ -80,7 +80,14 @@ module.exports = class SingleSpaReactGenerator extends Generator {
       // Extend with react-specific package json for typescript
       this.fs.extendJSON(
         this.destinationPath("package.json"),
-        this.fs.readJSON(this.templatePath("typescript/package.json"))
+        this.fs.readJSON(this.templatePath("typescript/package.json")),
+        (key, value) => {
+          if (key === "devDependencies") {
+            // Remove standard eslint configuration in favor of react specific
+            delete value["eslint-config-ts-important-stuff"];
+          }
+          return value;
+        }
       );
     }
   }

--- a/packages/generator-single-spa/src/react/templates/typescript/package.json
+++ b/packages/generator-single-spa/src/react/templates/typescript/package.json
@@ -1,5 +1,6 @@
 {
   "devDependencies": {
+    "eslint-config-ts-react-important-stuff": "^2.0.0",
     "webpack-config-single-spa-react-ts": "^1.4.2"
   }
 }

